app/models/audio_version.rb

```diff
@@ -48,6 +55,12 @@ def set_status
       return
     end
 
+    if !audio_formats_match?
+      self.status = INVALID
+      self.status_message = 'Mismatch in audio formats between audio files'
```

Comment:
Might be nice to indicate which thing(s) didn't match - or just the first thing.  "Audio format" is a bit ambiguous... I might confuse that for meaning "file 1 is mp3 and file 2 is wav".

Guessing we'll update the UI to display the freq/rate/channels for each file.  And maybe even highlight the mismatch in red.  So maybe I'm overthinking this part.